### DISCUSSION TITLE
Group hover transition-all not apply

### DISCUSSION THREAD

**Discussion by prpanto** [tailwindlabs/tailwindcss#10712] [Help] [answered]
Hello guys,
I have this little [project](https://play.tailwindcss.com/0LjxU6d6QX) and try to set transition with group hover but I dont know why it doesnt work. Appreciate for your help.

**Comment by wongjn** [comment, marked as answer]
Transitions don't "play" on the same frame as when `display` is changed from/to `none`. We can instead look at reworking the styling to avoid `display: none`. Here's an example that might be something like what you're after, in [this Tailwind Play](https://play.tailwindcss.com/VpXeT9VxHv).

**Reply by prpanto** [reply]
Thanks for this.
